library/modules/bigip_gtm_wide_ip.py

```diff
@@ -119,6 +119,42 @@
         module.
     type: str
     version_added: 2.8
+  persistence:
+    description:
+      - When C(enabled), ensures that when a local DNS makes repetitive requests on
+        behalf of a client, the system reconnects the client to the same resource
+        as previous requests.
+      - When C(disabled), ensures repetitive requests do not reconnect the client
+        to the same resource.
+    type: str
+    choices:
+      - disabled
+      - enabled
+    default: disabled
+    version_added: 2.9
+  persistence_ttl:
+    description:
+      - Specifies the time to maintain a connection between an local DNS and
+        a particular virtual server. The default setting is 3600 seconds.
+    type: int
+    default: 3600
+    version_added: 2.9
+  persist_cidr_ipv4:
+    description:
+      - Specifies a mask used to group IPv4 LDNS addresses. This feature
+        allows one persistence record to be shared by LDNS addresses
+        that match within this mask. The default is 32.
```

Comment:
we do not inform what the default is by device as it may change version to version

Reply:
There are other scenarios, many of them, where the default has been specified. Removing regardless.